.gitignore

```diff
@@ -0,0 +1,4 @@
+/obj
+bld*.inc
+build
```

Comment:
Some IDEs create another directories for build. Something like "build-Release", etc.

Reply:
this is not IDE specific